web/src/main/java/ca/hendriks/bartender/web/ingredient/IngredientsRestController.java

```diff
@@ -18,14 +18,26 @@ public IngredientsRestController(final IngredientRepository ingredientRepository
     }
 
     @GetMapping
+    @RequestMapping("/ingredients")
     public List<Ingredient> findAllIngredients() {
         return ingredientRepository.findAll();
     }
 
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
+    @RequestMapping("/ingredients/add")
```

Comment:
From [REST API URI Naming Conventions and Best Practices](https://restfulapi.net/resource-naming/)...
- RESTful URI should refer to a resource that is a thing (noun) instead of referring to an action (verb)
- We should not use URIs to indicate a CRUD function. URIs should only be used to identify the resources and not any action upon them uniquely.
- We should use HTTP request methods to indicate which CRUD function is performed.
- If we use verbs in the URI, we are most probably creating an RPC-style method call having a JSON or XML request/response format. It would be incorrect to call it REST.

Reply:
I know this but.. it feels super weird to have URIs that are just plain

Reply:
ya?